changelog.d/2-features/non-admin-commits

```diff
@@ -0,0 +1 @@
+Allow non-admins to commit add proposals in MLS conversations
```

Comment:
And remove proposals too.

Reply:
Remove proposals were already working fine.